pep-0514.txt

```diff
@@ -200,12 +240,16 @@ always named ``InstallPath``, and the default value must match ``sys.prefix``::
 If a string value named ``ExecutablePath`` exists, it must be a path to the
 ``python.exe`` (or equivalent) executable. Otherwise, the interpreter executable
 is assumed to be called ``python.exe`` and exist in the directory referenced by
-the default value.
+the ``(Default)`` value.
 
 If a string value named ``WindowedExecutablePath`` exists, it must be a path to
 the ``pythonw.exe`` (or equivalent) executable. Otherwise, the windowed
 interpreter executable is assumed to be called ``pythonw.exe`` and exist in the
-directory referenced by the default value.
+directory referenced by the ``(Default)`` value.
+
+Tools should be able to handle the case where the referenced executables do not
+actually exist, particularly when values for ``ExecutablePath`` and
+``WindowedExecutablePath`` are inferred from the ``(Default)`` value.
 
```

Comment:
That suggestion is basically identical to what is there now, except it adds an extra burden on a Python distribution to include a command-line entry point.

The Python distro for [Windows IoT Core](https://github.com/ms-iot/python/releases) doesn't have a normal entry point, as it is supposed to be deployed to a remote device. Given the choice between "I can register this without an InterpreterCommand" and "I have to invent some new way to register this", I'd rather people chose the first and tools learned to deal with it. (This becomes more important when there are multiple side-by-side versions of such a package.)

You may also want to register the install location of Python embedded in Blender or another app, since there's a working standard library and (IIRC) a python**.dll that could be used.

Perhaps InterpreterCommand should not have a fallback (except for PythonCore <= 3.5). Tools that need to be able to launch a process can simply ignore those. (I don't want to get into narrow feature registration -- that's a PEP of its own.)